enginetest/queries/queries.go

```diff
@@ -6965,6 +6965,14 @@ var QueryTests = []QueryTest{
 		Query:    "with recursive a as (select 1 union select 2) select * from (select 1 where 1 in (select * from a)) as `temp`",
 		Expected: []sql.Row{{1}},
 	},
+	{
+		Query: "create table test (id int primary key, first_name varchar(MAX))",
+		Expected: []sql.Row{
+			{
+				sql.OkResult{},
+			},
+		},
+	},
```

Comment:
@fulghum is this test is correct? I'm not quite sure since this is the first test with `CREATE` command

Reply:
I ran this test with `enginetest/memory_engine_test.go` with function `TestQueries(...)` and it already passed

Reply:
Hey @rafimuhammad01, sorry for the slow response – I was OOTO yesterday. I tested out this query with go-mysql-server on main and got the syntax error about `MAX` that I expected. Do you have a `go.mod` replace defined that's causing your local Vitess updates to be included? That's the only thing I could think of why this would be passing for you. 

I'll take a look through the tests and see if there's a better place to put this test. 

Could you merge main into this branch when you get a chance? Looks like there's a conflict with a change to `queries.go` on main that we should resolve.

Reply:
I poked around a little more and the `enginetest/queries/create_table_queries.go` is a better place for a test for this column definition form. The test format is slightly different in that file, but it should be super straightforward to add a new test for `varchar(max)` there. Let me know if you do hit any snags though and we're happy to help figure it out.

Reply:
btw... I just merged the Vitess change into go-mysql-server's main branch, so once you merge in changes from main, you shouldn't need a replace override in `go.mod` to point to your Vitess parser changes.

Reply:
hi @fulghum I mean the test passed after my implementation is done, I think this test is correct since it's failed before my implementation. I will change the location of the test then we can merge this PR

Reply:
Ahh... good! Thanks for clarifying!